### ISSUE TITLE
Hope linux support!

### ISSUE THREAD

**Issue by KuiperBelt-LAW** [wasabia/flutter_gl#57]
I used your flutter_gl in my projects,It works well , even it support web , amazing,
but It`s a pity about linux support, yes ,it `s important to multiplatform aim.  facing same question to others people I think.
Do you have a plan to strengthen it?

**Comment by forthtemple** [comment]
I agree. I have attempted to add Linux functionality to flutter_gl myself. I have attempted to merge the code in flutter_gl_windows and https://github.com/alnitak/flutter_opengl which has Linux functionality. My version compiles and runs but instead of the flutter example showing a yellow triangle with a red background it is completely black. I was wondering what I might be doing wrong and was hoping for some feedback. My repository is at:

https://github.com/forthtemple/flutter_gl

and my Linux code is in flutter_gl/linux in this repository.
